packages/dart/sshnoports/bin/sshnp.dart

```diff
@@ -136,6 +167,9 @@ void main(List<String> args) async {
           shell.stderr.listen(stderr.add);
           stdin.listen(shell.stdin.add);
 
+          // Cancel the previous ctrl-c listener which would call exitProgram
+          await sigintListener.cancel();
+
           // catch local ctrl-c's and forward to remote
           ProcessSignal.sigint.watch().listen((signal) {
             shell.stdin.add(Uint8List.fromList([3]));
```

Comment:
have to cancel the previous ctrl-c listener because now we're in the ssh shell session, we want to transmit ctrl-c's to the remote shell